lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -169,6 +169,7 @@ export default TestModule.extend({
       Ember.run(function() {
         Ember.setProperties(context, hash);
       });
+      return hash;
```

Comment:
Mostly that the 1.x behavior was not good, and undefined seems better than letting folks rely on the context being returned only to have that be a breaking change for them when they update to 2.x.